src/from-attribute.js

```diff
@@ -48,14 +48,12 @@ module.exports = function fromAttribute (propertyName) {
 			// Child binding used by `attributeChangedCallback` to update the value when an attribute change occurs 
 			const childValue = value.to(instance, propertyName);
 			const bind = new Bind({
-				parent: value.from(instance.getAttribute(propertyName)),
+				parent: value.from(instance.getAttribute(propertyName) || undefined),
 				child: childValue,
 				queue: "dom",
-				setChild (newVal) {
-					// During initialization prevent update of child when parent attribute does not exist
-					if (instance.hasAttribute(propertyName)) {
-						canReflect.setValue(childValue, newVal);
-					}
+				setChild () {
```

Comment:
Is this empty `setChild` needed?

Reply:
@phillipskevin When the bindings are initialized within the mixins:
```js
var initializeData = stacheBindings.behaviors.initializeViewModel(bindings, props, (properties) => {
  super.initialize(properties);
  return this;
}, bindingContext);
```
This sets the properties on the instance, when the `binding.start()` is called calls `setChild` internally which then also set's the property on the instance, so I was seeing duplicate calls to the property setter. 

It was late last night and I think this can be changed to use: `onInitDoNotUpdateChild: true`